<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Playblast</class>
 <widget class="QWidget" name="Playblast">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Playblast</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Resolution</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="widthEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="heightEdit">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Frame range: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="frameStart"/>
       </item>
       <item>
        <widget class="QLineEdit" name="frameEnd"/>
       </item>
       <item>
        <widget class="QPushButton" name="fromTimeLineButton">
         <property name="text">
          <string>From timeline</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Size:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="sizeWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="size25">
            <property name="text">
             <string>25%</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="size50">
            <property name="text">
             <string>50%</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="size75">
            <property name="text">
             <string>75%</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="size100">
            <property name="text">
             <string>100%</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Output: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="margin">
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="formatQuicktime">
            <property name="text">
             <string>Quicktime</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="formatTiff">
            <property name="text">
             <string>Tiff</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Camera:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cameraCombo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="setViewButton">
       <property name="text">
        <string>Set view to camera!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Stereo options: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
        <item>
         <widget class="QRadioButton" name="stereoOff">
          <property name="text">
           <string>Off</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="stereoAna">
          <property name="text">
           <string>Anaglyph in Camera</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="stereoMulti">
          <property name="text">
           <string>Both Eyes</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Play with: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="playwithCombo">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Open when done</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Submit to Dailies: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Overscan 1,0:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="overscan">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Add sound if exists</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="soundCheck">
       <property name="text">
        <string/>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Show ornamens: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ornamentsCheck">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Output resolution: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="widthFinalEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="heightFinalEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="playBlastButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>45</height>
      </size>
     </property>
     <property name="text">
      <string>Playblast!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="playLatestButton">
     <property name="text">
      <string>Play latest!</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>size25</sender>
   <signal>clicked()</signal>
   <receiver>Playblast</receiver>
   <slot>updateOutputResolution()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>size50</sender>
   <signal>clicked()</signal>
   <receiver>Playblast</receiver>
   <slot>updateOutputResolution()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>472</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>size75</sender>
   <signal>clicked()</signal>
   <receiver>Playblast</receiver>
   <slot>updateOutputResolution()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>size100</sender>
   <signal>clicked()</signal>
   <receiver>Playblast</receiver>
   <slot>updateOutputResolution()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widthEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Playblast</receiver>
   <slot>updateOutputResolution()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>heightEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Playblast</receiver>
   <slot>updateOutputResolution()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateOutputResolution()</slot>
 </slots>
</ui>
